server/mailrepository/deleted-messages-vault-repository/pom.xml

```diff
@@ -1,94 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That is a separate task!

We are not dropping this as:

 - we need to be doing a migration
 - That is still, as of now, the only implementation working

Please revert this commit.